proftpd/map.jinja

```diff
@@ -1,10 +1,12 @@
-{% import_yaml "proftpd/defaults.yaml" as default_settings %}
+{% import_yaml "proftpd/defaults.yaml" 	as default_settings %}
+{% import_yaml "proftpd/sftp.yaml" 	as sftp_settings %}
```

Comment:
Is a different default file needed for `sftp` ou could it be merged in one unique file? Has it be done for clarity or for anything else?

Reply:
I merge default.yaml and sftp.yaml, it's more simple